pom.xml

```diff
@@ -60,6 +60,18 @@ limitations under the License.
       <artifactId>google-cloud-datastore</artifactId>
       <version>1.52.0</version>
     </dependency>
+    
+    <dependency>
+      <groupId>com.google.cloud</groupId>
+      <artifactId>google-cloud-language</artifactId>
+      <version>1.55.0</version>
+    </dependency>
+
+    <dependency>
```

Comment:
Maybe leave a comment explaining why we need this library?

Reply:
Added, to the best of my ability! I'm still a little confused about it myself, but hopefully this will help anyone else reading the dependency list later and wondering.